packages/react-aria-components/stories/Form.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {action} from '@storybook/addon-actions';
+import {Button, Form, Input, Label, ListBox, ListBoxItem, Popover, Select, SelectValue, TextField} from 'react-aria-components';
+import React from 'react';
+
+export default {
+  title: 'React Aria Components'
+};
+
+
+export const FormAutoFillExample = () => {
+  return (
+    <Form
+      aria-label="Shipping information"
+      onSubmit={e => {
+        action('onSubmit')(Object.fromEntries(new FormData(e.target as HTMLFormElement).entries()));
+        e.preventDefault();
+      }}>
+      <TextField>
+        <Label>Address</Label>
+        <Input name="streetAddress" type="text" id="streetAddress" autoComplete="shipping street-address" />
```

Comment:
weird, this one renders as autocomplete off for me...
https://reactspectrum.blob.core.windows.net/reactspectrum/df4ad8242241bf9959c9148d20b4be683ab320df/storybook/index.html?path=/story/react-aria-components--form-auto-fill-example&providerSwitcher-express=false
![Screenshot 2025-02-06 at 11 17 00 am](https://github.com/user-attachments/assets/1170927b-babc-4672-ab87-517a30db9d8a)

Reply:
huh strange. it looks like it has autocomplete=""shipping address-level2" based on the screenshot. does it not autofill for you either?

Reply:
ah, node above that one

Reply:
huh, well, i updated chrome last night. now it's rendering. My Chrome autofill works on it, but my Dashlane doesn't recognize it for some reason still. At this point though, I think it's correct and I'm just running into local issues